cmd/pj-rehearse/main.go

```diff
@@ -281,21 +283,22 @@ func rehearseMain() int {
 	metrics.RecordOpportunity(toRehearseClusterProfiles, "cluster-profile-change")
 	toRehearse.AddAll(toRehearseClusterProfiles)
 
-	rehearsals := rehearse.ConfigureRehearsalJobs(toRehearse, prConfig.CiOperator, prNumber, loggers, o.allowVolumes, changedTemplates, changedClusterProfiles)
-	metrics.RecordActual(rehearsals)
-	if len(rehearsals) == 0 {
+	jobConfigurer := rehearse.NewJobConfigurer(toRehearse, changedPeriodics, prConfig.CiOperator, prNumber, loggers, o.allowVolumes, changedTemplates, changedClusterProfiles)
+	presubmits, periodics := jobConfigurer.ConfigureRehearsalJobs()
+	metrics.RecordActual(presubmits)
```

Comment:
Metrics will need to be done properly. I think the current fields should be used for totals (all rehearsals opportunities, all rehearsals taken) and new fields should be added for how many of these were presumits, periodics etc.